### DISCUSSION TITLE
Convert connected non-*.dds textures and into *.dds at export

### DISCUSSION THREAD

**Discussion by ssnd292** [StjerneIdioten/I3D-Blender-Addon#104] [Ideas & Feature Requests]
To always have uptodate textures, and automatic DDS conversion of connected non-DDS textures could be added later down the road.

Wopster made a conversion tool that could be supplied with the toolset eventually

The lookup to know which texture needs which compression could be based on a suffix system
_decals = DXT5
_spec = DXT1
_normal = DXT1 
... eca

**Comment by kbrandwijk** [comment]
If you get to this: please make that possible to turn that off (especially the normals).

**Comment by StjerneIdioten** [comment]
That wouldn't be a problem to implement. Although I don't see why it would be a problem or at least for FS19 I know that it bitches in the console if your textures aren't dds :-P

**Comment by kbrandwijk** [comment]
> _normal = DXT1

Building normals can also be DXT5, as they support an alpha channel for height maps.
